### ISSUE TITLE
[Infra] Deploy DevNet

### ISSUE THREAD

**Issue by okdas** [pokt-network/poktroll#23]
## Objective

We need to deploy DevNets (remote LocalNets) somewhere in the cloud.

## Goals

- Unblock remote e2e testing;
- Provide an ability for developers to create ephemeral and more permanent DevNets to test their changes;

## Deliverables

- [ ] Connect with Grove infra team to get options considering costs, and existing tooling;
- [ ] Provision infra or re-use existing Grove infrastructure to create DevNets;
- [ ] Reuse Helm chart from https://github.com/pokt-network/helm-charts;
- [ ] Cleanup resources that are no longer necessary due to sunsetting v1 infra in https://github.com/pokt-network/protocol-infra

## Non-goals / Non-deliverables

- No need to deploy full-blown UI and dashboards

## General deliverables

- [ ] **Comments**: Add/update TODOs and comments alongside the source code so it is easier to follow.

---

**Creator**: @okdas

**Comment by okdas** [comment]
Closed as we actually had https://github.com/pokt-network/poktroll/issues/17 for this work.

**Comment by Olshansk** [comment]
Adding the `duplicate` label
